templates/components/challenge_area.html

```diff
@@ -48,7 +48,11 @@
   </div>
 </div>
 
-<script type="text/javascript">
+<script type="module">
+  import PassedState from "{{ url_for('static', filename='js/passed-state.js')}}";
+
```

Comment:
Can we move `passedState = new PassedState();` to passed-state.js, and import `passedState`?